### PR TITLE
Fix `add-tag-to-commits` and `mark-merge-commits-in-list`

### PR DESCRIPTION
**Pull request by yakov116**
Test on https://github.com/sindresorhus/refined-github/commits/master/.gitattributes

### CODE REVIEW

File: source/features/mark-merge-commits-in-list.tsx [hunk 1]
```diff
@@ -36,7 +36,11 @@ const filterMergeCommits = async (commits: string[]): Promise<string[]> => {
 // eslint-disable-next-line import/prefer-default-export
 export function getCommitHash(commit: HTMLElement): string {
 	return 	commit.dataset.channel!.split(':')[3] ?? // Pre "Repository refresh" layout
-	commit.querySelector<HTMLAnchorElement>('a[href]')!.href.split('/').pop()!;
+	commit.querySelector<HTMLAnchorElement>('a[href]')!
+		.href
+		.split('/')
+		.pop()!
+		.split('#')[0]; // Drop the #diff-;
```
Comment (kidonng) [inline source/features/mark-merge-commits-in-list.tsx:43]: Maybe you can use `pathname` so you don't have to split again?

```suggestion
	commit.querySelector<HTMLAnchorElement>('a[href]')!.pathname.split('/').pop()!;
```

Comment (yakov116) [reply source/features/mark-merge-commits-in-list.tsx]: Yeah that's better

**Comment by yakov116** [comment]
In case your waiting for me to test it. I tested it and it works.

File: source/features/add-tag-to-commits.tsx [hunk 2]
```diff
@@ -137,7 +140,10 @@ async function init(): Promise<void | false> {
 			// There was no tags for this commit, save that info to the cache
 			commitsWithNoTags.push(targetCommit);
 		} else if (targetTags.length > 0) {
-			select('.commit-meta', commit)!.append(
+			select([
+				'.commit-meta', // Pre "Repository refresh" layout
+				'p + div.d-flex'
```
Comment (yakov116) [inline source/features/add-tag-to-commits.tsx:145]:
```suggestion
p + div.d-flex:not(.Details-content--hidden)
```

If someone wants to send a pr. If not i can later tonight

**Comment by kidonng** [comment]
This completely stopped working when I was writting https://github.com/sindresorhus/refined-github/pull/3342#issuecomment-663003308, but it works fine except there's still error.

**Pull request merged**